services/src/main/scala/cromwell/services/metadata/MetadataService.scala

```diff
@@ -57,18 +55,17 @@ object MetadataService {
     extends ReadAction
   case class GetMetadataQueryAction(key: MetadataQuery) extends ReadAction
   case class GetStatus(workflowId: WorkflowId) extends ReadAction
-  case class WorkflowQuery(uri: Uri, parameters: Seq[(String, String)]) extends ReadAction
+  case class WorkflowQuery[A](uri: A, parameters: Seq[(String, String)]) extends ReadAction
```

Comment:
Dunno. The whole attempt at an RFC 5988 implementation is broken in multiple ways, so I was touching as little as possible to get the web bits out of the business logic.